### PR TITLE
breaking(zipkin): removes servicName from zipkin exporter.

### LINKED ISSUE #1549: Zipkin exporter should take service.name from Resource
Zipkin requires a service name to be sent with each span.  Per the specification https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#service-name , 

    Zipkin service name MUST be set to the value of the resource attribute: service.name. 
    If no service.name is contained in a Span's Resource, it MUST be populated from the default Resource

### PR DESCRIPTION
**Pull request by jcchavezs**
Resource detector provides a serviceName in all cases, hence we can relay on span resource to obtain the serviceName. Also this is required by the spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md\#service-name 

Closes #1549

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `"go.opentelemetry.io/sdk/metric/controller.basic".WithPusher` is replaced with `WithExporter` to provide consistent naming across project. (#1656)
 - Added non-empty string check for trace `Attribute` keys. (#1659)
 - Add `description` to SpanStatus only when `StatusCode` is set to error. (#1662)
+- Removes `serviceName` parameter from Zipkin exporter and uses resource instead. (#1549)
```
Comment (bogdandrutu) [inline CHANGELOG.md:25]: Not sure about the standard, but I see all using "Remove" here :)

**Review by bogdandrutu — APPROVED** [review]
My LGTM does not count, but I like giving it 👍

**Review by XSAM — CHANGES_REQUESTED** [review]
Looks good. But we should update `example/zipkin/main.go` as well.

**Comment by jcchavezs** [comment]
Done.

**Review by XSAM — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jcchavezs) [reply CHANGELOG.md]: Done.

**Pull request merged**